### ISSUE TITLE
DG- Architecture section

### ISSUE THREAD

**Issue by varnika1402** [varnika1402/pe#11]
Overall the architecture section is too wordy and complex even when only a high-level explanation of architecture based on the diagram is needed.

**Comment by nus-se-script** [comment]
[IMPORTANT!: Please do not edit or reply to this comment using the GitHub UI. You can respond to it using CATcher during the next phase of the PE]

# Team's Response
In our DG, we cater to people who need to understand with more explanation and wording. If you prefer not to use it, you may skip to the appropriate sections that help you. Some people need them.

## The 'Original' Bug

[The team marked this bug as a duplicate of the following bug]

># DG- implementation section too wordy
>
>**Note from the teaching team:** This bug was reported during the _Part II (Evaluating Documents)_ stage of the PE. **You may reject this bug if it is not related to the quality of documentation.**<hr>
>The implementation section of each feature mentioned in DG is too wordy. A simple class diagram and sequence diagram could have been used to show implementation of each feature instead along with steps to describe the implementation. 
>
>
>
>
>
><hr>
><sub>[original: nus-cs2113-AY2223S2/pe-interim#2418] [original labels: severity.Low type.DocumentationBug]</sub>
>
>
>

## Their Response to the 'Original' Bug

[This is the team's response to the above 'original' bug]

>The design of the program is quite complex and the developers decide that it is appreciated to many more that we include the step by step procedure. Readers who prefer to read at a different level of abstraction (e.g., not to read specifically step by step) can just skip to the sequence diagrams or just the architectures, or in this case any point of the documentation. This can be sped up using the content list as well. Our DG was designed to cater to lower denominators in this regard who prefer to read and view the diagrams in a lot more precision and detail.

# Items for the Tester to Verify

## :question: Issue duplicate status

Team chose to mark this issue as a duplicate of another issue (as explained in the _**Team's response**_ above)

- [ ] I disagree

**Reason for disagreement:** [replace this with your reason]

<catcher-end-of-segment><hr>

## :question: Issue response

Team chose [`response.Rejected`]

- [ ] I disagree

**Reason for disagreement:** [replace this with your reason]

<catcher-end-of-segment><hr>

## :question: Issue severity

Team chose [`severity.Low`]
Originally [`severity.VeryLow`]

- [ ] I disagree

**Reason for disagreement:** [replace this with your reason]

<catcher-end-of-segment><hr>
